tests/e2e/README.md

```diff
@@ -96,6 +96,20 @@ make push
 
 ```
 
+**Option 1.5:** Build your own images and use an in-cluster registry
+
+```
+# Set up env vars in .istiorc.mk
+cat .istiorc.mk
+LOCALREG=true
+HUB="10.10.0.2:5000"
+TAG=latest
+```
+# Setting the LOCALREG var will notify the e2e test targets to bring up the local registry within cluster and
+# build/push images onto it.
+
+# After the test is done, the registry will be cleanup by default.
```

Comment:
How not to clean up the images and registry?

Reply:
If flag `skip-cleanup` is specified.